doc/API.md

```diff
@@ -0,0 +1,335 @@
+110,990 vbytes of transactions paying between 38 and 53 sat/vB, 138,976 vbytes paying between 34 and 38, etc.
+
+
+### `GET /mempool/txids`
+
+Get the full list of txids in the mempool as an array.
+
+The order of the txids is arbitrary and does not match bitcoind's.
+
+### `GET /mempool/recent`
+
+Get a list of the last 10 transactions to enter the mempool.
+
+Each transaction object contains simplified overview data, with the following fields: `txid`, `fee`, `vsize` and `value`
+
+## Fee estimates
+
+### `GET /fee-estimates`
+
+Get an object where the key is the confirmation target (in number of blocks)
+and the value is the estimated feerate (in sat/vB).
+
+The available confirmation targets are 1-25, 144, 504 and 1008 blocks.
+
+For example: `{ "1": 87.882, "2": 87.882, "3": 87.882, "4": 87.882, "5": 81.129, "6": 68.285, ..., "144": 1.027, "504": 1.027, "1008": 1.027 }`
+
+## Tokens
+
+### `GET /color/:color_id`
+
+Get information about a colored coin.
+
+Returns an object with:
+
+- `color_id`
+- `chain_stats` and `mempool_stats`, each with:
+  - `tx_count`
+  - `issuance_count`
+  - `issued_amount`
+  - `transfer_count`
+  - `transferred_amount`
+  - `burn_count`
+  - `burned_amount`
+
+Example :
+
+```
+{
+  "color_id": "c16f0279e9ed041c3d710a9f57d0c02928416460c4b722ae3457a11eec381c526d",
+  "chain_stats": {
+    "tx_count": 54,
+    "issuance_count": 2,
+    "issued_amount": 1600000000,
+    "transfer_count": 1,
+    "transferred_amount": 600000000,
+    "burn_count": 0,
+    "burned_amount": 0
+  },
+  "mempool_stats": {
+    "tx_count": 3,
+    "issuance_count": 0,
+    "issued_amount": 0,
+    "transfer_count": 0,
+    "transferred_amount": 0,
+    "burn_count": 1,
+    "burned_amount": 100000000
+  }
+}
+```
+
+### `GET /color/:color_id/txs`
+### `GET /color/:color_id/txs/mempool`
+### `GET /color/:color_id/txs/chain[/:last_seen]`
+
+Get transactions associated with the specified color coin.
+
+Returns a list of issuance, reissuance, transfer and burn transactions.
+
+The total number of results will be returned as the `x-total-results` header.
+
+## Transaction format
+
+- `txid`
+- `version`
+- `locktime`
+- `size`
+- `weight`
+- `fee`
+- `vin[]`
+  - `txid`
+  - `vout`
+  - `is_coinbase`
+  - `scriptsig`
+  - `scriptsig_asm`
+  - `inner_redeemscript_asm`
+  - `sequence`
+  - `prevout` (previous output in the same format as in `vout` below)
+- `vout[]`
+  - `scriptpubkey`
+  - `scriptpubkey_asm`
+  - `scriptpubkey_type`
+  - `scriptpubkey_address`
+  - `value`
+- `status`
+  - `confirmed` (boolean)
+  - `block_height` (available for confirmed transactions, `null` otherwise)
+  - `block_hash` (available for confirmed transactions, `null` otherwise)
+  - `block_time` (available for confirmed transactions, `null` otherwise)
+
+## Block format
+
+- `id`
+- `height`
+- `version`
+- `timestamp`
+- `tx_count`
+- `size`
+- `weight`
+- `merkle_root`
+- `im_merkle_root`
+- `previousblockhash`
+- `mediantime` (median time-past)
```

Comment:
```suggestion
- `time`
```

Reply:
Field `timestamp` should be renamed to `time`?